package.json

```diff
@@ -8,8 +8,8 @@
     "test": "tests"
   },
   "scripts": {
-    "e2e": "playwright test",
-    "e2e:report": "playwright show-report",
+    "pwtests": "docker run --rm --network host -v $(pwd):/work/ -w /work/ -it mcr.microsoft.com/playwright:v1.35.1-jammy /bin/bash -c \"npm ci; npx playwright install; npx playwright test\"",
```

Comment:
Comment that the version of the image here has to exactly match the versions of @playwright/test in devDependencies and the Github Actions configuration. The version in devDependencies should just be "1.35.1" without the `^`, so it doesn't auto-upgrade and get out of sync. You should probably remove `playwright-webkit` from "dependencies" entirely.

You also should avoid running `npx playwright install`: the image includes that data, and we want this to fail quickly if the versions get out of sync, instead of doing the slow download of all the browser versions.

The commands should probably be separated with `&&` instead of `;` so that later ones don't run if earlier ones fail.

Finally, make sure that all of those docker flags are actually needed, and probably spell out their --long-forms instead of the single-character option names. https://docs.docker.com/storage/bind-mounts/ argues to replace -v with --mount.

Reply:
Comments in json are not allowed, so I added a pwtests-comment command :).  
Need to look up -it, --rm, -w, and -c to find their long-form names.